### PR TITLE
fix(visitor_statistic_image): fix path error with DirectoryIterator

### PR DESCRIPTION
**Pull request by skai92**
The visitor_statistic_image threw errors while loading the language file. This way the language files will be loaded correctly (See https://github.com/webSPELL/webSPELL/blob/dev/admin/admincenter.php)
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/Phhere%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/pull/246%23issuecomment-107999202%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-06-02T15%3A57%3A03Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20b1fa113d3a8ed8eac79491417edb5b750fba9747%20admin/visitor_statistic_image.php%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/pull/246%23discussion_r30511911%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20../%20can%20not%20be%20correct%20because%20together%20with%20chdir%28../%29%20it%20would%20result%20in%20going%20one%20folder%20to%20far%22%2C%20%22created_at%22%3A%20%222015-05-18T14%3A28%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22Aw%20sorry%20for%20that.%20That%5Cu00b4s%20right.%22%2C%20%22created_at%22%3A%20%222015-05-18T14%3A30%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1869584%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/skai92%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20admin/visitor_statistic_image.php%3AL25-36%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/webSPELL/webSPELL/pull/246%23issuecomment-107999202%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/246%23discussion_r30511911%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/246%23discussion_r30512144%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/Phhere'></a>

- [ ] <a href='#crh-comment-Pull b1fa113d3a8ed8eac79491417edb5b750fba9747 admin/visitor_statistic_image.php 5'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/pull/246#discussion_r30511911'>File: admin/visitor_statistic_image.php:L25-36</a></b>
- <a href='https://github.com/Phhere'></a> This ../ can not be correct because together with chdir(../) it would result in going one folder to far
- <a href='https://github.com/skai92'></a> Aw sorry for that. That´s right.

<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/246?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/246?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/246?approve=0'></a>&nbsp;<a href='https://github.com/webSPELL/webSPELL/pull/246'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by webSPELL-org** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/webSPELL/webSPELL/pull/246'>Refresh</a>.<a href='#crh-update-none'></a>

File: admin/visitor_statistic_image.php [hunk 1]
```diff
@@ -25,9 +25,11 @@
 ##########################################################################
 */
 
+chdir('../');
 include("../_mysql.php");
```
Comment (Phhere) [inline admin/visitor_statistic_image.php]: This ../ can not be correct because together with chdir(../) it would result in going one folder to far

Comment (skai92) [inline admin/visitor_statistic_image.php]: Aw sorry for that. That´s right.

**Comment by Phhere** [comment]
:shipit:

**Pull request merged**